### PR TITLE
fix: Production hot fix to install gunicorn

### PR DESCRIPTION
**Pull request by gwhitney**
The python package gunicorn is only needed in production (but it is required in that setting). Hence there is nothing in requirements.txt etc that installs it. This means that when .venv is blown away (such as migrating to a new server, recovering from a failure, or simply testing full installation or cleaning out an old .venv that has accumulated unneeded code), production fails unless gunicorn is installed. (And indeed, that is exactly what happened when #109 was installed in production.) This PR adds a line to server/production.sh to install gunicorn, and precisely reflects the hot fix that is already present and running in production to cope with the situation.

In other words, merging this and installing in production will result in no change to current production, and should prevent future such failures when .venv has to be reset.

### CODE REVIEW

**Comment by Vectornaut** [comment]
This looks reasonable to me. We now effectively have two sets of requirements: the core requirements in`requirements-freeze.txt`, and the additional production requirements in the subsequent lines of the production startup script. From that perspective, should we specify a version for `gunicorn`, like we do for each of the core requirements?

**Review by Vectornaut — APPROVED** [review]
This looks reasonable to me. We now effectively have two sets of requirements: the core requirements in`requirements-freeze.txt`, and the additional production requirements in the subsequent lines of the production startup script. From that perspective, should we specify a version for `gunicorn`, like we do for each of the core requirements?

**Comment by katestange** [comment]
It works on my setup.  I will merge.

**Comment by gwhitney** [comment]
Successfully installed in production after wiping out .venv and creating a fresh one. Appears to be working normally.
